### ISSUE TITLE
sort filter options ?

### ISSUE THREAD

**Issue by daslicht** [Shopify/liquid#770]
Hi,
where to find all parameters which we can pass to the sort filter ? please?

~Marc

**Comment by parkr** [comment]
I don't believe it takes an arguments. Docs: http://shopify.github.io/liquid/filters/sort/

If you're using Jekyll, see our docs: http://jekyllrb.com/docs/templates/

**Comment by daslicht** [comment]
```sort: 'title'``` works, but only ascending and case sensitive :/

**Comment by kainjow** [comment]
There is sort_natural which is case insensitive. It appears to be undocumented. Use reverse filter to change the sort order.
